apis/apiextensions/apiextensions.go

```diff
@@ -31,6 +32,7 @@ func init() {
 		v1.AddToScheme,
 		v1beta1.AddToScheme,
 		v1alpha1.AddToScheme,
+		v2alpha1.AddToScheme,
```

Comment:
@negz is it clear to you how basic reconciliation of XRDs was working without this? 🤔

Reply:
Yeah - the XRD controller still watches and reconciles v1 XRDs. So until now there wasn't actually any Crossplane controller code that cared about v2alpha1. Only the API server knew about that version.

I imagine once we release v2.0 proper I'll switch the code to use the new v2 type. I didn't yet mostly because it felt strange to update the code to use the new type while it's marked alpha.